packages/aws-cdk-lib/aws-route53-targets/lib/props.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Properties the alias record target
+ */
+export interface IAliasRecordTargetProps {
+  /**
+   * Evaluate target health
+   *
+   * @default - no health check configuration
+   */
+  readonly evaluateTargetHealth?: boolean;
+}
```

Comment:
nit: Can you name the file `shared.ts` instead of `props`.

Reply:
hi @GavinZZ just renamed the file, please check it out